index.js

```diff
@@ -69,3 +69,33 @@ btnClear.addEventListener('click', function () {
     console.log(nr1);
 
 });
+
+// Please let me know if I should move the following 
+// variables, on the top. 
+
+let btnSubtraction = document.querySelector('#btn-subtraction');
+let btnAddition = document.querySelector('#btn-addition');
+let btnDivision = document.querySelector('#btn-division');
+let btnMultiplication = document.querySelector('#btn-multiplication');
+
+let operation;
+
+btnSubtraction.addEventListener('click', function () {
+    operation = '-';
+    console.log(operation);
+});
+
+btnAddition.addEventListener('click', function () {
+    operation = '+';
+    console.log(operation);
+})
+
+btnDivision.addEventListener('click', function () {
+    operation = "÷";
```

Comment:
Let's use the "/" character so it's the same one we would use when doing division in JS.